core/src/main/java/org/apache/spark/api/java/function/DoubleFunction.java

```diff
@@ -23,5 +23,5 @@
  *  A function that returns Doubles, and can be used to construct DoubleRDDs.
  */
 public interface DoubleFunction<T> extends Serializable {
-  public double call(T t) throws Exception;
+  double call(T t) throws Exception;
```

Comment:
I will willingly exclude any changes (including the interface methods) here if needed.